README.md

```diff
@@ -159,25 +99,20 @@ What do we need to know?
 Make sure to have this information before adding it to our configuration.
 
 1. Add a new section to [`foodsoft/app_config.yml`](foodsoft/app_config.yml). You could copy the
-   `demo` instance. Make sure that each foodcoop has a unique identifier, and doesn't contain
-   any 'weird' characters. You may set the _name_ as well. The database should be lowercase alphabet,
-   prefixed with `foodsoft_` (in this example that is `foodsoft_myfoodcoop`). Make sure to set it in
-   the configuration.
+   `demo` instance. Make sure that each foodcoop has a unique identifier that matches `^[a-z]+$`.
```

Comment:
I think we have foodcoops with a dash, underscore and/or digits, and that should be ok, right?

Reply:
Yes, you are right. I missed the [one foodcoop with a digit](https://github.com/foodcoops/foodcoops.net/blob/bf709612c7c9735de44b1319885824cc5d1e6031/foodsoft/app_config.yml#L313).

There are no dashes/underscores, though. My approach here is: Let us not allow untested symbols. It is easier to loosen strict requirements (if necessary) than it is to repair broken installs. I have experienced in the past that rails transforms strings and symbols between underscores/camelcase/hyphenation. The foodcoop identifier is also used in the database name, so I really prefer to keep it simple.

Since digits seem to work fine, we could add them. Do you agree that `^[a-z0-9]+$` is a good choice?